src/main/java/com/couchbase/lite/Database.java

```diff
@@ -1038,9 +1034,7 @@ public void removeDatabaseListener(DatabaseListener listener) {
     @InterfaceAudience.Private
     public List<Replication> getActiveReplications() {
         List<Replication> activeReplicatorsList = new ArrayList<Replication>();
-        if (activeReplicators != null) {
-            activeReplicatorsList.addAll(activeReplicators);
-        }
+        activeReplicatorsList.addAll(activeReplicators);
```

Comment:
It's not clear to me if this operation is thread-safe even though activeReplicators is synchronized hash set. Can you confirm?